pkg/ext-proc/handlers/request.go

```diff
@@ -18,25 +18,29 @@ import (
 // HandleRequestBody handles body of the request to the backend server, such as parsing the "model"
 // parameter.
 // Envoy sends the request body to ext proc before sending the request to the backend server.
-func (s *Server) HandleRequestBody(reqCtx *RequestContext, req *extProcPb.ProcessingRequest) (*extProcPb.ProcessingResponse, error) {
-	klogV := klog.V(logutil.VERBOSE)
-	klogV.InfoS("Handling request body")
+func (s *Server) HandleRequestBody(
+	logger logr.Logger,
```

Comment:
It would be more pretty to pass around `context.Context` as a rule and load the logger, but this would really add quite a few logger lookup calls. So you will see I am passing `logr.Logger` around where a `context.Context` might be more pretty. I am open to suggestions, though.

Reply:
Is it really expensive to lookup the logger? 

btw, I think the standard is to pass context as the first parameter.

Reply:
Yeah, I know. Amended code to pass a ctx.